### PR TITLE
Fixes bug in utils.string.stripTags for Windows 7 + IE10

### PR DESCRIPTION
**Pull request by jeroendesloovere**
When trying to add new "editor" block in Pages module then you'll previously got error.

### CODE REVIEW

File: src/Backend/Core/Js/utils.js [hunk 1]
```diff
@@ -303,7 +304,13 @@ utils.string =
...
 	stripTags: function(value)
 	{
-		return value.replace(/<[^>]*>/ig, '');
+		if(typeof value === 'undefined') return '';
+		else
+		{
+			value = value.replace(/<[^>]*>/ig, '');
+		}
+
+		return value;
```
Comment (WouterSioen) [inline src/Backend/Core/Js/utils.js]: I would restructure this code block this way:

```js
if (typeof value === 'undefined') {
    return '';
}

return value.replace(/<[^>]*>/ig, '');
```

This has exactly the same outcome, but is clearer imo.

**Comment by jeroendesloovere** [comment]
Don't know if it has something to do with this...

**new bug**

### Steps to reproduce

1. Log in to /private + production environment 
2. Modules > Pages > a page > add editor block
3. Add some text
4. Click ok
=> then you don't see any text in the editor block.

schermafbeelding 2016-03-04 om 09 31 48
+ When saving the editor blocks aren't saved

**Comment by WouterSioen** [comment]
:+1:

**Comment by jeroendesloovere** [comment]
@WouterSioen Seems my fix above is improving things to go to the next step (from clicking "add new" to showing the editor). But then another bug pops up when clicking "OK"... Editor visualBlock is empty.

**Comment by WouterSioen** [comment]
If I understood the discussion correctly, this fixes a notice but introducers a critical bug?

**Comment by jeroendesloovere** [comment]
Closed since updating ```matthiasmullie/minify``` to version 1.3.34 fixed the problem.
See #1429 for more information.

**Pull request closed without merging**